cloud/wfm-rest-api/src/impl/MongoDbRepository.ts

```diff
@@ -13,7 +14,10 @@ const dbError: ApiError = new ApiError(errorCodes.DB_ERROR, 'MongoDbRepository d
 /**
  * Service for performing data operations on mongodb database
  */
-export class MongoDbRepository<T extends { id: string }> implements PagingDataRepository<T> {
+export class MongoDbRepository<T extends {
+  id: string,
+  _id?: string | ObjectID
```

Comment:
When this repository is grabbing the data from mongo it comes with `_id: ObjectID('...')`, but it's serialized to a string when we try to save it again and mongo complains that it won't change it.

We can either `delete data._id` like I did or try to wrap it again in an `ObjectID` constructor and try to update it back to the same value. I had some trouble trying out the second option.